src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const sumCost = days * 40;
```

Comment:
better rename more meaningful way.
```suggestion
  const pricePerDay = 40;
  const totalPrice = pricePerDay * days;
```

Reply:
Fixed!